### PR TITLE
new branch to realese routeSetup

### PR DESCRIPTION
**Pull request by Zelenro**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,5 +1,5 @@
 API_URL=http://localhost:3000
 PORT=3000
-DB_CONNECTION_STRING=''
+DB_CONNECTION_STRING=postgresql://obox_admin:eomkyby1nvXktZC5U7fLlfShLh9S64cA@dpg-cprhs6bqf0us738fmp30-a.frankfurt-postgres.render.com/obox_fpqx?ssl=true
```
Comment (y-andriievskyi) [inline .env.example:3]: this is sensitive, things like that should never be pushed to repository. store it in your local .env file only

File: ui/package.json [hunk 2]
```diff
@@ -10,18 +10,20 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@vitejs/plugin-react-swc": "^3.7.0",
     "axios": "^1.6.8",
     "dotenv": "^16.4.5",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
```
Comment (y-andriievskyi) [inline ui/package.json:17]: lock versions when you add something. Remove ^ before package version

File: ui/src/layouts/AuthLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+
+import {Outlet} from 'react-router-dom'
+
+const AuthLayout =()=>{
```
Comment (y-andriievskyi) [inline ui/src/layouts/AuthLayout.tsx:4]: Incorrect formatting.
Check MainLayout.tsx

File: ui/src/routes/AuthRoute.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const AuthRoute =()=>{
```
Comment (y-andriievskyi) [inline ui/src/routes/AuthRoute.tsx:1]: Formatting

File: ui/src/routes/MainRoute.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+const MainRoute =()=>{
```
Comment (y-andriievskyi) [inline ui/src/routes/MainRoute.tsx:1]: Formatting

File: ui/src/routes/NotFoundRoute.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+const NotFoundRoute =()=>{
```
Comment (y-andriievskyi) [inline ui/src/routes/NotFoundRoute.tsx:1]: Formatting

**Review by y-andriievskyi — CHANGES_REQUESTED** [review]

**Review by Zelenro — COMMENTED** [review]
correct commit - clear sensitive information

**Pull request closed without merging**